### ISSUE TITLE
How to see the metrics of a code-gen trained model (using the .py code)

### ISSUE THREAD

**Issue by CESARDELATORRE** [Azure/automl-codegen-preview#5]
Once the model is trained, if getting the metrics by code, you get:
![image](https://user-images.githubusercontent.com/1712635/129421734-73246d9b-a35b-46a2-9d3c-e0648f9e3139.png)

And if using the UI, it's empty:
![image](https://user-images.githubusercontent.com/1712635/129421777-147e7e75-84d7-4947-8581-2d338b30512a.png)

We need to explain the user how to see the metrics if the metrics were generated.. I cannot find the code where the metrics are generated in the .py, though... so we might be missing that part in the code-gen and that's why there are no metrics?

**Comment by wchill** [comment]
Yes, currently the code to calculate the metrics is missing. This is being tracked internally

**Comment by CESARDELATORRE** [comment]
@wchill , @rtanase , this is something we need to discuss, if we'll implement a feature/task for the validation metrics in GenCode or not... by PUBLIC PREVIEW release

The thing is that if the user provided a Validation dataset or specified a train/validation split in %, we could easily generate the code for calculating the validation metrics by performing predictions, etc.

But if the user provided a single dataset, we'd need to do cross-validation and that would change the generated code... it wouldn't be just a single .fit() but more complexity on the generated training code and probably we might have additional dependencies with other AutoML packages... 

Thoughts?

**Comment by rtanase** [comment]
@CESARDELATORRE, let's see the feedback on this. I believe we discussed this a few months back and we decided to have the generated code for the CV case match what the generated model does (train on everything). That makes the code much simpler.
